x-pack/plugins/security_solution/public/common/components/event_details/cti_details/threat_details_view.tsx

```diff
@@ -76,8 +76,18 @@ const ThreatDetailsViewComponent: React.FC<{
   enrichments: CtiEnrichment[];
   showInvestigationTimeEnrichments: boolean;
   loading: boolean;
+  /**
+   * Slot to render something before the beforeHeader
```

Comment:
I would add to this comment that this was introduced because we're using this component in 2 places (the new place being the expandable flyout) and will soon be removed (once we go GA with the expandable flyout)
Could even write a TODO on top of the comment